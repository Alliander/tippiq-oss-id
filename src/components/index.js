/**
 * Point of contact for component modules.
 * @module components
 * @example import { Field, RegisterForm } from '../../components';
 */

export AccountMenu from './AccountMenu/AccountMenu';
export AuthenticateBox from './AuthenticateBox/AuthenticateBox';
export AutoRegisterForm from './AutoRegisterForm/AutoRegisterForm';
export AutoRegisterPrefilled from './AutoRegisterPrefilled/AutoRegisterPrefilled';
export ChangeUserNameForm from './ChangeUserNameForm/ChangeUserNameForm';
export ChangeEmailForm from './ChangeEmailForm/ChangeEmailForm';
export ChangePasswordForm from './ChangePasswordForm/ChangePasswordForm';
export CookieWall from './CookieWall/CookieWall';
export Field from './Field/Field';
export Footer from './Footer/Footer';
export LoginForm from './LoginForm/LoginForm';
export NavBar from './NavBar/NavBar';
export PageTitle from './PageTitle/PageTitle';
export PlaceKeyForm from './PlaceKeyForm/PlaceKeyForm';
export PrivacyLabel from './PrivacyLabel/PrivacyLabel';
export RegisterForm from './RegisterForm/RegisterForm';
export RequestResetPasswordForm from './RequestResetPasswordForm/RequestResetPasswordForm';
export ServiceProviderBanner from './ServiceProvider/Banner';
export ServiceProviderName from './ServiceProvider/Name';
export SetupPasswordAndNameForm from './SetupPasswordAndNameForm/SetupPasswordAndNameForm';
export Spinner from './Spinner/Spinner';
