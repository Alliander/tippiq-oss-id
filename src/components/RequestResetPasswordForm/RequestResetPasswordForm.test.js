import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { RequestResetPasswordForm } from '../../components';

test('Basic render snapshot test', () => {
  const component = shallow(<RequestResetPasswordForm
    fields={{ password: 'password' }}
    handleSubmit={() => {}}
    errorMessage={'errormessage'}
  />);

  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});

