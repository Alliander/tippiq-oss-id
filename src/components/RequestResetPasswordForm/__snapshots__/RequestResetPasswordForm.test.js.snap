// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic render snapshot test 1`] = `
<ReduxFormConnector(RequestResetPasswordForm)
  destroyOnUnmount={true}
  errorMessage="errormessage"
  fields={
    Object {
      "password": "password",
    }
  }
  form="request-reset-password"
  getFormState={[Function]}
  handleSubmit={[Function]}
  reduxMountPoint="form"
  submitPassback={[Function]}
  touchOnBlur={true}
  touchOnChange={false}
  validate={[Function]}
/>
`;
